app/Http/Livewire/TodayLecturerCard.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Livewire;
```

Comment:
Untuk class livewirenya kita pakai struktur folder gini coba mas:
```
app/Http/Livewire/PublicHome/TodayLecturingSchedules.php
```
```suggestion
namespace App\Http\Livewire\PublicHome;
```